### PR TITLE
Update search logic on get

### PR DESCRIPTION
**Pull request by pyakushin**

### CODE REVIEW

File: src/core/cluster/quorum.rs [hunk 1]
```diff
@@ -250,37 +274,36 @@ impl Cluster for Quorum {
...
-        let second_attempt = Self::get_all(key, &sup_nodes, GetOptions::new_alien()).await;
-        debug!("GET[{}] cluster ans sup: {:?}", key, second_attempt);
+        let local_node = self.mapper.local_node_name();
+        let target_nodes = self
+            .mapper
+            .nodes()
+            .iter()
+            .filter(|node| node.name() != local_node);
+        // @TODO don't wait for all answers, return on first success
+        let results = Self::get_all(key, target_nodes, GetOptions::new_alien()).await;
```
Comment (agend) [inline src/core/cluster/quorum.rs:290]: we already sent request for ALL before. do we need to query alien directly?

File: src/core/cluster/quorum.rs [hunk 2]
```diff
@@ -231,14 +231,38 @@ impl Cluster for Quorum {
...
+            Err(e) => error!("local node backend returned error: {}", e),
+        };
+        trace!("GET[{}] ~~~LOOKUP REMOTE NODES~~~", key);
+        let local_node = self.mapper.local_node_name();
+        let target_nodes = self
+            .get_target_nodes(key)
+            .iter()
+            .filter(|node| node.name() != local_node);
+        // @TODO don't wait for all answers, return on first success
+        let results = Self::get_all(key, target_nodes, GetOptions::new_all()).await;
```
Comment (agend) [inline src/core/cluster/quorum.rs:265]: Should it have force option as well as we set particular nodes list?

File: src/core/cluster/quorum.rs [hunk 3]
```diff
@@ -113,7 +113,7 @@ impl Quorum {
```
Comment (agend) [inline src/core/cluster/quorum.rs:113]: We should also rename quorum to smething else - like simple_cluster_operations

File: src/core/cluster/quorum.rs (hunk 2 above)
Comment (pyakushin) [reply src/core/cluster/quorum.rs]: [outdated] `force_node` flag is in `Operation` and is true in all backend operations after cluster

File: src/core/cluster/quorum.rs (hunk 1 above)
Comment (pyakushin) [reply src/core/cluster/quorum.rs]: fixed

**Pull request merged**